public/pages/DetectorResults/containers/AnomalyResults.tsx

```diff
@@ -313,7 +313,10 @@ export function AnomalyResults(props: AnomalyResultsProps) {
   };
 
   const getInitProgressMessage = () => {
-    return detector && isDetectorInitializing && detector.initProgress
+    return detector &&
+      isDetectorInitializing &&
+      detector.initProgress &&
```

Comment:
Use lodash get ?

Reply:
will use it

Reply:
good idea, probably many places within this page and other places in the plugin where we should change to this to make it simpler